lib/MusicBrainz/Server/Controller/Role/Alias.pm

```diff
@@ -49,6 +52,14 @@ sub aliases : Chained('load') PathPart('aliases')
         entity => $entity->TO_JSON,
     );
 
+    if ($ENTITIES_WITH_RELATIONSHIP_CREDITS{$self->{entity_name}}) {
```

Comment:
Assuming `$self->{entity_name}` comes from `Controller::Role::Load`, I'm not sure that will always work, e.g. we use `rg` for release groups. `$entity->entity_type` should work.